external/anomaly/anomaly_classification/configs/configuration.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
Should we also add what this file holds?

Reply:
I agree, doctstring missing. I guess these all because we're missing our tox and pre-commit stuff here